src/app/icd/ICDManager.cpp

```diff
@@ -33,19 +33,28 @@
 #define ICD_ENFORCE_SIT_SLOW_POLL_LIMIT 0
 #endif
 
+#ifndef ICD_REPORT_ON_ENTER_ACTIVE_MODE
+// Enabling this makes the devive emmit subscription reports when transitionning from idle to active mode. It gives the device a
+// chance to sleep for the extent of its idle mode interval before getting awakened to emit reports again.
```

Comment:
I guess I feel like this is not accurately describing the actual behavior implication here. We could allow "sleep for the extent of its idle mode interval before getting awakened to emit reports again" even if we did not force-emit on every wakeup, as long as we force-emitted when the next wakeup would be sooner than the idle mode interval.

That is the behavior we are implementing is _sufficient_ for allowing said sleep, but it's not _necessary_.

Reply:
Understood, second sentence removed.